maps/biter_battles_v2/difficulty_vote.lua

```diff
@@ -155,10 +155,14 @@ local function on_gui_click(event)
         return
     end
 	
-	game.print(player.name .. " has voted for " .. difficulties[i].name .. " difficulty!", difficulties[i].print_color)
-	global.difficulty_player_votes[player.name] = i
-	set_difficulty()
-	difficulty_gui_all()
+	if not (global.difficulty_player_votes[player.name] == i) or global.difficulty_player_votes[player.name] == nil then
```

Comment:
```suggestion
	if global.difficulty_player_votes[player.name] ~= i then
```
Can it be done like this, or is there some catch with nil?